sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -55,7 +55,8 @@ private[hive] object IsolatedClientLoader extends Logging {
       sharedPrefixes: Seq[String] = Seq.empty,
       barrierPrefixes: Seq[String] = Seq.empty): IsolatedClientLoader = synchronized {
     val resolvedVersion = hiveVersion(hiveMetastoreVersion)
-    // We will use Hadoop 2.7 if we cannot resolve the Hadoop artifact.
+    // We will use hadoop-3.3.1 if we can't resolve the hadoop artifact
+    // when builtin hadoop is Hadoop 3. Otherwise we will use hadoop-2.4.7.
```

Comment:
`Otherwise we will use hadoop-2.4.7.` -> `Otherwise we will use Hadoop 2.x`.

Reply:
Done